aslam_cv_pipeline/src/visual-npipeline.cc

```diff
@@ -193,38 +206,42 @@ std::shared_ptr<const NCamera> VisualNPipeline::getOutputNCameras() const {
 }
 
 size_t VisualNPipeline::getNumFramesProcessing() const {
-  std::unique_lock<std::mutex> lock(mutex_);
+  std::lock_guard<std::mutex> lock(mutex_);
   return processing_.size();
 }
 
-void VisualNPipeline::work(size_t camera_index, const cv::Mat& image, int64_t timestamp) {
+void VisualNPipeline::work(size_t camera_index, const cv::Mat& image,
+                           int64_t timestamp) {
```

Comment:
`timestamp_nanoseconds`

Reply:
maybe this shows up at other places too in this class... if you can make the units clear, it'd be great. if it's too much of a hassle, leave it.